content/docs/rfcs/57/README.md

```diff
@@ -364,7 +364,7 @@ Operators can freely choose how they want to generate, and distribute the public
 
 The following concepts are introduced:
 * `private-key-topic`: A private key of 32 bytes, that allows the holder to sign messages and it's mapped to a `protected-pubsub-topic`.
-* `app-message-hash`: Application `WakuMessage` hash, calculated as `sha256(concat(pubsubTopic, payload, contentTopic))` with all elements in bytes.
+* `app-message-hash`: Application `WakuMessage` hash, calculated as `sha256(concat(pubsubTopic, payload, contentTopic, timestamp, ephemeral))` with all elements in bytes.
```

Comment:
maybe it can be removed, since how `app-message-hash` is specified below. so we have just 1 source of truth.